src/processing/script/transformers/location-get.ts

```diff
@@ -68,6 +68,11 @@ const transformer: Transformer<Identifier> = {
         if (parent.type === Syntax.MethodDefinition)
             return false;
 
+        // Skip: class X { location }
+        // @ts-ignore
+        if (parent.type === Syntax.PropertyDefinition)
```

Comment:
I'm happy to add this to `esotope-hammerhead` if there is consensus on merging this.